.travis.yml

```diff
@@ -6,12 +6,11 @@ python:
 install: 
     - sudo apt-get install libmysqlclient-dev
     - pip install -r requirements.txt
+    - pip install codecov pytest-cov
 
 script:
-    - coverage run --source propertysuggester -m pytest -v
-    - flake8 . --max-line-length=100
```

Comment:
I wonder why the line length setting is gone?

Reply:
because the default is 80 (and PEP8 says so too) and I reduced the maximum line to that number so it's not needed anymore.